source/standards/source-code/working-with-git.html.md.erb

```diff
@@ -142,37 +142,28 @@ commands like git merge and git revert.
 
 ## Branching/merging conventions
 
-You may often choose to work on a particular feature on a "feature branch"
-rather than directly on `main`. Indeed, given how cheap branches are in Git,
-[using branches](http://git-scm.com/book/en/Git-Branching-Basic-Branching-and-Merging) is positively encouraged.
-
-You are encouraged to make liberal use of Git's [history rewriting
-features](http://git-scm.com/book/en/Git-Tools-Rewriting-History) while working
-locally, in order to arrange your commits into appropriate logical chunks that
-will make sense to your fellow developers. In particular, you may find
-`git rebase --interactive` very useful. You are also encouraged to avoid merge
-commits and use `git rebase main` instead. However, you should not rewrite commits
-that have been pushed unless you:
-
-  * are very sure that no-one else will be affected by you rewriting the
-    branch history
-  * have an extremely good reason. For example: someone has committed
-    sensitive information (personally identifiable data, passwords and suchlike)
-    and it needs purging from history
-
-When in doubt you should err towards smaller commits, which can be rebased
-together later. It's harder to break large commits out into smaller chunks.
-
-The smaller commits should still be logical chunks, but this will give context
-for a more specific change and make git tools like `annotate` and `log` more
-useful.
-
-When merging from a feature branch to main (or any other mainline development
-branch), in particular one that has previously been shared with colleagues, you
-should use `git merge`'s `--no-ff` option to preserve evidence of your feature
-branch in the repository history. This advice may be freely ignored for smaller
-local feature branches for which a fast-forward merge will look like any other
-routine development work on `main`.
+You should develop on a short-lived "feature" [branch](https://git-scm.com/book/en/Git-Branching-Basic-Branching-and-Merging) rather than directly on the trunk branch, which is usually `main` or `master`, unless there is a compelling reason to do otherwise.
+
+For example if your repository has many binary files, the `git diff` will not provide useful change information, and you may choose to collaborate directly on the trunk branch.
+
+You should prefer staging your commits as small chunks rather than having a single commit per pull-request.
```

Comment:
I think there's a lot of scope for people (particularly those who are new to working with VCS/DVCS) to misinterpret "small chunks". I think we can be a bit more specific and accurate while conveying more about the underlying intent:

```suggestion
You should prefer organising your commits as small, self-contained changes rather than having a single commit per pull-request. Ideally, each commit should represent a coherent set of changes.
```